samples/matter/light_switch/README.rst

```diff
@@ -12,6 +12,7 @@ This light switch sample demonstrates the usage of the :ref:`Matter <ug_matter>`
 When configured together with the :ref:`Matter light bulb <matter_light_bulb_sample>` sample (or other lighting sample) and when using a Matter controller, the light switch can control one light bulb directly or a group of light bulbs remotely over a Matter network built on top of a low-power, 802.15.4 Thread, or on top of a Wi-Fi network.
 Support for both Thread and Wi-Fi is mutually exclusive and depends on the hardware platform, so only one protocol can be supported for a specific light switch device.
 In case of Thread, this device works as a Thread :ref:`Sleepy End Device <thread_ot_device_types>`.
+In case of Wi-Fi, this device works in ``legacy power save mode`` that means the device sleeps most of the time, and wakes up on each Delivery Traffic Indication Message (DTIM) interval to receive a beacon message.
```

Comment:
```suggestion
In case of Wi-Fi, this device works in ``legacy power save mode`` that means the device sleeps most of the time, and wakes up on each Delivery Traffic Indication Message (DTIM) interval to receive a beacon message.
```
"to receive a beacon message" is not the whole point of this operation as far as I understand the concept.. "to poll for incoming messages" would be more accurate. The beacon with DTIM flag is just used to indicate that now it's a right moment for the device to send the poll request.

Reply:
Right, we wake up periodically based on how frequent the beacons are being sent and poll only in case the beacon's TIM contains the proper AID.